docs/web-console-guide_5/master.adoc

```diff
@@ -90,7 +153,35 @@ include::topics_5/web-report-static.adoc[leveloffset=+2]
 ////
 
 // Using Custom Rules
-include::topics_5/web-custom-rules.adoc[leveloffset=+1]
+//include::topics_5/web-custom-rules.adoc[leveloffset=+1]
+
+== Using global rules and labels
+
+Global rules and labels apply to all projects and are automatically applied to analyses for which they fit the definitions of a project, unless they are explicitly excluded in the *Advanced options* of the project. The {ProductShortName} includes a preconfigured set of global rules and labels and you can define your own custom global rule and labels.
+
```

Comment:
That last sentence needs to be broken up because the custom global rules (not rule) and labels gets lost.
Put the "custom global rules and labels into its own sentence and paragraph. 

I'm trying to see whether your section is actually about "Using" custom rules/labels or "Creating".